scripts/integtest.sh

```diff
@@ -98,9 +98,12 @@ then
   leader=$(echo  $data | cut -d ',' -f1 | cut -d ':' -f1,2 )
   follower=$(echo $data  |  cut -d ',' -f2 | cut -d ':' -f1,2 )
   
+  # Get number of nodes, assuming both leader and follower have same number of nodes
+  numNodes=$((${follower##*:} - ${leader##*:}))
```

Comment:
Fine for now but this logic can break is all nodes do not get ports assigned in sequence.

Reply:
The topology class in opensearch-build always assigns port sequentially.